packages/code-snippet/src/CodeSnippetWidget.tsx

```diff
@@ -16,11 +16,16 @@
 
 import '../style/index.css';
 
+import { PythonFileEditor } from '@elyra/python-runner-extension/lib/PythonFileEditor';
```

Comment:
@karlaspuldaro I just noticed this imported wrong, make sure to fix this in your next commit (prob will need to add it as a dependency in package.json too